models/commons/src/high-precision-money/high-precision-money-draft/generator.ts

```diff
@@ -0,0 +1,16 @@
+import { fake, Generator, oneOf } from '@commercetools-test-data/core';
+import { THighPrecisionMoneyDraft } from '../types';
+
+// https://docs.commercetools.com/api/types#highprecisionmoneydraft
+
+const generator = Generator<THighPrecisionMoneyDraft>({
+  fields: {
+    type: 'highPrecision',
+    centAmount: fake((f) => f.number.int({ min: 1, max: 999999 })),
```

Comment:
Similar situation here, we should either set it to null or compute a valid rounded version. Or it will throw an error when we submit as is:

> A Price of 1.015 USD can be rounded either to 1.01 USD or 1.02 USD. If it lies outside of this range, an error message stating that centAmount must be rounded correctly will be returned. 

https://docs.commercetools.com/api/types#highprecisionmoneydraft